icon_validator/rules/plugin_validators/runtime_validator.py

```diff
@@ -0,0 +1,72 @@
+import os
+import glob
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class RuntimeValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_setup(spec):
+        if "setup.py" in os.listdir(spec.directory):
+            with open(f"{spec.directory}/setup.py", "r") as file:
+                setup_str = file.read().replace("\n", "")
+
+                if f'install_requires=["insightconnect-plugin-runtime"]' not in setup_str\
+                        and f"install_requires=['insightconnect-plugin-runtime']" not in setup_str:
+                    raise ValidationException("Komand is no longer used for install_requires in setup.py. "
+                                              "Use insightconnect-plugin-runtime instead.")
+
+    @staticmethod
+    def validate_imports(spec):
+        for root, dirs, files in os.walk(spec.directory):
+            for file in files:
+                if file.endswith(".py"):
+                    with open(os.path.join(root, file), "r") as open_file:
+                        file_str = open_file.read().replace("\n", "")
+
+                        if "import komand" in file_str or "from komand" in file_str:
```

Comment:
Sorry, I missed it earlier. We have  imports like below for which we don't want to flag.

`from komand_abuseipdb.util import helper` 
`import komand_eml.util.utils as utils`

Reply:
Good catch!